### PR TITLE
Initial commit - CICA dev

### PR DESCRIPTION
**Pull request by armoj**

### CODE REVIEW

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/cica-apply-for-compensation-dev/01-rbac.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cica-apply-for-compensation-dev-admin
+  namespace: cica-apply-for-compensation-dev
+subjects:
+  - kind: Group
+    name: "github:criminalInjuriesCompensationAuthority"
```
Comment (kalbir) [inline namespaces/cloud-platform-live-0.k8s.integration.dsd.io/cica-apply-for-compensation-dev/01-rbac.yaml]: Hi @armoj -- at the moment we only give access to teams that sit under the `ministryofjustice` github organisation. Are you able to get a CICA team setup under that org?

Comment (armoj) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/cica-apply-for-compensation-dev/01-rbac.yaml]: There is already a CICA team within the MinistryOfJustice GH. I have requested to be a member.

Comment (armoj) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/cica-apply-for-compensation-dev/01-rbac.yaml]: I am now a member of the CICA team in the MoJ org. Is this what was needed from me?

Comment (kalbir) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/cica-apply-for-compensation-dev/01-rbac.yaml]: @armoj You just need to change that team name to match the one in the MoJ github org. In this case it is:

```suggestion
    name: "github:CICA"
```

**Comment by razvan-moj** [comment]
Hi @armoj PR is fine but won't get applied automatically, please do a squash&merge - the ns will be created seconds later

**Comment by SteveMarshall** [comment]
@armoj Do you still want to set up this namespace?

**Comment by armoj** [comment]
This PR is no longer required as @Adesanluj is working on this particular task at the moment. He has requested read/write access to the MoJ GH org in order to carry out this task for CICA. I understand there are ongoing discussions around this request. Muqadas Butt, and Joel Samual have discussed it on slack here: https://mojdt.slack.com/archives/C52LPHPK8/p1543239394215700.

Is this something that you can help get sorted quickly? We are eager to get things started within the cloud platform dev work.

**Pull request closed without merging**